core/sequencer/sequencer.go

```diff
@@ -329,9 +329,3 @@ func (s *Sequencer) createEpoch(ctx context.Context, d *domain.Domain, logClient
 	glog.Infof("CreatedEpoch: rev: %v, root: %x", mapRoot.Revision, mapRoot.RootHash)
 	return nil
 }
-
-// TODO(gdbelvin): Add leaf at a specific index. trillian#423
-func queueLogLeaf(ctx context.Context, logClient *tclient.LogClient, smr *tpb.SignedMapRoot) error {
-	// Queue the leaf and then wait until it has been sequenced and verified.
-	return logClient.AddLeaf(ctx, smr.MapRoot)
```

Comment:
For a potential follow-up PR: Maybe `AddLeafAndWait` would be less a misleading name than `AddLeaf`? For a moment I thought "Why does the new code waits and the old one doesn't?", but then I realized the old one did wait.

Reply:
yep. I'll make that change in Trillian